OpenEdXMobile/src/main/java/org/edx/mobile/view/BaseWebViewFragment.java

```diff
@@ -82,6 +84,10 @@ public void onDestroy() {
     }
 
     private void initWebView() {
+        // To make the smooth scrolling of view within viewpager2
+        // Inspiration: https://stackoverflow.com/a/35198484
+        ViewCompat.setNestedScrollingEnabled(webView, true);
```

Comment:
IMO it doesn't seem to work properly on webview

Reply:
Removed, not work properly.

Reply:
basically I meant to say this line isn't effecting the behavior